### PR TITLE
Improve asValue

### PR DESCRIPTION
**Pull request by swansontec**
Built on #22.

### CODE REVIEW

File: src/cleaners/asValue.js [hunk 1]
```diff
@@ -1,8 +1,15 @@
-export function asValue(value) {
+export function asValue(...values) {
+  let message = 'Expected '
+  if (values.length !== 1) message += 'one of: '
+  for (let i = 0; i < values.length; ++i) {
+    if (i > 0) message += ' | '
+    message += JSON.stringify(values[i])
+  }
```
Comment (samholmes) [inline src/cleaners/asValue.js]: Can we do this lazily? Or perhaps just move it to line 13. I feel like we should only be computing the error message when the cleaner fails.

File: docs/reference.md [hunk 2]
```diff
@@ -182,6 +183,34 @@ const asMaximum = asOptional(asNumber)
 asMaximum(null) // returns undefined
 ```
 
+## asValue
```
Comment (samholmes) [inline docs/reference.md:186]: Mention "enum" somewhere in this section for the noobs.

File: test/cleaners/asValue.test.ts [hunk 3]
```diff
@@ -4,31 +4,53 @@ import { describe, it } from 'mocha'
...
 describe('asValue', function () {
+  const asTrue = asValue(true)
   const as123 = asValue(123)
   const asFoo = asValue('foo')
   const asNull = asValue(null)
   const asUndefined = asValue(undefined)
-  const asTrue = asValue(true)
+  const asEnum = asValue(undefined, true, 'too-hot', 'too-cold', 42)
+
+  it('accepts boolean values', function () {
```
Comment (samholmes) [inline test/cleaners/asValue.test.ts:14]: _accepts_... I always get those words mixed up. 👍

**Review by samholmes — CHANGES_REQUESTED** [review]

File: src/cleaners/asValue.js (hunk 1 above)
Comment (swansontec) [reply src/cleaners/asValue.js]: Moved to the later area, as suggested.

File: docs/reference.md (hunk 2 above)
Comment (swansontec) [reply docs/reference.md:186]: Ok, I re-wrote it a bit.

**Review by samholmes — APPROVED** [review]

**Pull request merged**